packages/engine/src/scene/functions/loaders/InstancingFunctions.ts

```diff
@@ -72,8 +77,14 @@ export const GRASS_PROPERTIES_DEFAULT_VALUES: GrassProperties = {
   bladeHeight: { mu: 0.3, sigma: 0.05 },
   bladeWidth: { mu: 0.03, sigma: 0.01 },
   joints: 4,
-  grassTexture: 'https://resources-dev.etherealengine.com/assets/grass/blade_diffuse.jpg',
-  alphaMap: 'https://resources-dev.etherealengine.com/assets/grass/blade_alpha.jpg',
+  grassTexture: {
+    src: 'https://resources-dev.etherealengine.com/assets/grass/blade_diffuse.jpg',
```

Comment:
We should be sourcing these from the repo rather than the engine in case they are unreachable

Reply:
https://github.com/XRFoundation/XREngine/issues/7605